packages/api/src/types.ts

```diff
@@ -1,13 +1,46 @@
 import { Static, Type, TSchema } from '@sinclair/typebox'
 export { Db, Collection, ObjectId, WithId } from 'mongodb'
 
-const Nullable = <T extends TSchema>(type: T) => Type.Union([type, Type.Null()])
+export const Nullable = <T extends TSchema>(type: T) =>
+  Type.Union([type, Type.Null()])
 
 export const ClaimPlayerParams = Type.Object({
   key: Type.String(),
 })
 export type ClaimPlayerParams = Static<typeof ClaimPlayerParams>
 
+// Socials
+
+export const Socials = Nullable(
```

Comment:
I think that the the base type shouldn't be `Nullable` because if `socials` exists always would be non `null`. Could we make `Nullable` where we are using this runtime type?